utils/actions/markdownHelpers/count.ts

```diff
@@ -11,8 +11,10 @@ const countMarkdown = ({
     if (count.error) {
       return `\n We're sorry, we ran into an error: ${count.error}`;
     } else {
-      if (count.name && count.github_app_uses) {
+      if (count.name && (count.github_app_uses <= 500)) {
         return `\n Your team ${count.name} has used Watermelon ${count.github_app_uses} times.`;
+      } else {
+        return `\n Your team has surpassed the free monthly usage. [Please click here](https://calendly.com/evargas-14/watermelon-business) to upgrade.`;
```

Comment:
Do we still want to show the full comment?

Reply:
You mean the `Your team ${count.name} has....` comment?

Reply:
I mean, not showing anything until paid?

Reply:
Ahh yes good point. I'll change that

Reply:
Fixed